documentation/uploading_to_cpg.md

```diff
@@ -105,4 +105,15 @@ Run your transfer commands to `staging-cellpainting-gallery`.
 
 Once the transfers are complete, either you (Imaging Platform internal) or your data champion (if external) must verify the data transferred to `staging-cellpainting-gallery` is complete.
 (Currently this is done manually, though this will be programatic in the future.)
+
+To verify if the transfer was succesful you (Imaging Platform internal) can compare:
```

Comment:
```suggestion
To verify if the transfer was successful, compare object counts between your source and destination.
Because of differences in the way file sizes are calculated between file systems and object storage, file size is not a reliable metric for comparison.
```